pkg/api/converterfromapi.go

```diff
@@ -1265,7 +640,6 @@ func convertAzureEnvironmentSpecConfigToVLabs(api *AzureEnvironmentSpecConfig, v
 	vlabses.KubernetesSpecConfig = vlabs.KubernetesSpecConfig{
 		AzureTelemetryPID:                api.KubernetesSpecConfig.AzureTelemetryPID,
 		KubernetesImageBase:              api.KubernetesSpecConfig.KubernetesImageBase,
-		MCRKubernetesImageBase:           api.KubernetesSpecConfig.MCRKubernetesImageBase,
```

Comment:
ditto

Reply:
that's really strange. is that something new? I did not remove these specifically, so I'm betting on a rebase, or the merge preview?

Reply:
fixed and squashed into the offending commit